db/migrate/20200331150436_rename_foreman_features.rb

```diff
@@ -0,0 +1,35 @@
+class RenameForemanFeatures < ActiveRecord::Migration[5.1]
+  class MiqProductFeature < ActiveRecord::Base; end
+  class MiqRolesFeature < ActiveRecord::Base; end
+
+  def up
+    return if MiqProductFeature.none?
+
+    MiqProductFeature.find_by(:identifier => 'provider_foreman_explorer')&.update(:identifier => 'ems_configuration')
```

Comment:
I think we should do `update!` as opposed to `update` here and elsewhere in the file.